prometheus_client/registry.py

```diff
@@ -18,6 +18,14 @@ def collect(self) -> Iterable[Metric]:
         return []
 
 
+class DuplicateTimeseries(ValueError):
+    def __init__(self, duplicates):
+        msg = 'Duplicated timeseries in CollectorRegistry: {}'.format(
+            duplicates)
+        super().__init__(msg)
+        self.duplicates = duplicates
```

Comment:
Is the idea here that a client of this library can see what the duplicate names are? If so I think some typing would be nice here as it is for external consumption.

Reply:
That's a fair point. I've added the typing definition.